test/AutoDiff/protocol_requirement_autodiff.swift

```diff
@@ -181,4 +181,16 @@ public protocol MissingDifferentiableDistribution: DifferentiableDistribution
   func logProbability(of value: Value) -> Float // expected-note {{candidate is missing attribute '@differentiable(wrt: self)'}}
 }
 
+// Missing `@differentiable` attribute, without printing the 'wrt' arguments.
+protocol Example {
+  associatedtype Scalar
+
+  @differentiable(where Scalar: Differentiable)
```

Comment:
This case is malformed. We should reject any `where` clause in `@differentiable` attributes.

Reply:
Oh I see. I'll remove that then. However, we should probably add the check for this in a separate PR.

Reply:
I removed the malformed part but left the rest of the test.